### PR TITLE
Client - Added baseToken to node info response

### PR DESCRIPTION
**Pull request by matjazv**
# Description of change

Updated node info response `deserialization` and `print` functions.

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How the change has been tested

Unit tests in `test_get_info.c` passed.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests using CTest that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/client/api/restful/get_node_info.c [hunk 1]
```diff
@@ -254,25 +263,97 @@ int deser_node_info(char const *const j_str, res_node_info_t *res) {
...
+
+  // gets base token unit
+  if ((ret = json_get_string(base_token_obj, JSON_KEY_UNIT, res->u.output_node_info->base_token.unit,
+                             sizeof(res->u.output_node_info->base_token.unit))) != 0) {
+    printf("[%s:%d]: gets %s json string failed\n", __func__, __LINE__, JSON_KEY_UNIT);
+    goto end;
+  }
+
+  // gets base token subunit
+  if ((ret = json_get_string(base_token_obj, JSON_KEY_SUBUNIT, res->u.output_node_info->base_token.subunit,
```
Comment (oopsmonk) [inline src/client/api/restful/get_node_info.c]: `subuint` would be omitted if null

File: tests/client/api_restful/test_get_info.c [hunk 2]
```diff
@@ -27,8 +27,8 @@ void test_get_info() {
     TEST_ASSERT(info->u.output_node_info->msg_per_sec >= 0.0);
     TEST_ASSERT(info->u.output_node_info->referenced_msg_per_sec >= 0.0);
     TEST_ASSERT(info->u.output_node_info->referenced_rate >= 0.0);
-    TEST_ASSERT(info->u.output_node_info->latest_milestone_timestamp > 390326400);
-    TEST_ASSERT(info->u.output_node_info->confirmed_milestone_index > 0);
+    // TEST_ASSERT(info->u.output_node_info->latest_milestone_timestamp > 390326400);
+    // TEST_ASSERT(info->u.output_node_info->confirmed_milestone_index > 0);
```
Comment (oopsmonk) [inline tests/client/api_restful/test_get_info.c]: remove

**Review by oopsmonk — CHANGES_REQUESTED** [review]

File: src/client/api/restful/get_node_info.c (hunk 1 above)
Comment (matjazv) [reply src/client/api/restful/get_node_info.c]: Thank you for finding this issue.

File: tests/client/api_restful/test_get_info.c (hunk 2 above)
Comment (matjazv) [reply tests/client/api_restful/test_get_info.c]: Dead code was removed. Thank you.

**Review by rebinmannady93 — APPROVED** [review]

**Review by oopsmonk — APPROVED** [review]

**Pull request merged**